sdk/documentintelligence/azure-ai-documentintelligence/tests/test_dmac_copy_model.py

```diff
@@ -67,13 +67,12 @@ def test_copy_model_successful(self, client, documentintelligence_storage_contai
 
         assert copy.api_version == model.api_version
         assert copy.model_id != model.model_id
+        assert copy.model_id == copy_auth["targetModelId"]
         assert copy.description == model.description
-        assert copy.created_date_time == model.created_date_time
-        assert copy.expiration_date_time == model.expiration_date_time
```

Comment:
Were these properties removed or are we just not testing them anymore?

Reply:
They are not removed, just not testing. Also, the created time and expiration time in copy are not always equivalent to the ones in original. So they're not necessary to check.